File: OnJeong_BE/src/main/java/com/a503/onjeong/domain/group/dto/GroupDTO.java
package com.a503.onjeong.domain.group.dto;

import com.a503.onjeong.domain.user.dto.UserDTO;
import lombok.Builder;
import lombok.Data;
import java.util.ArrayList;
import java.util.List;

@Data
public class GroupDTO {
    private Long groupId;
    private Long ownerId;
    private String name;
    private List<UserDTO> userList;

    @Builder
    public GroupDTO(
            Long groupId,
            Long ownerId,
            String name,
            List<UserDTO> userList
    ) {
        this.groupId = groupId;
        this.name = name;
        this.ownerId = ownerId;
        this.userList = userList;
    }
    public void updateOwnerId (Long ownerId) {
        this.ownerId = ownerId;
    }
    public void updateName (String name) {
        this.name = name;
    }

}

File: OnJeong_FE/app/src/main/java/com/a503/onjeong/domain/mypage/dto/GroupDTO.kt
package com.a503.onjeong.domain.mypage.dto


import lombok.Builder
import lombok.Data


@Data
class GroupDTO @Builder constructor(
     val groupId: Long,
    private val ownerId: Long,
     val name: String,
    userList: List<UserDTO>
) {
     var userList: List<UserDTO> = ArrayList()
    init {
        this.userList = userList
    }
}


